packages/bitcore-node/src/models/transaction.ts

```diff
@@ -25,6 +25,7 @@ export type ITransaction = {
   fee: number;
   size: number;
   locktime: number;
+  raw: string;
```

Comment:
This `raw` value should be part of the `_apiTransform` return object so that the API actually returns it.